### PR TITLE
Add update-motd.d

### PR DESCRIPTION
**Pull request by QuentinPerez**

### CODE REVIEW

**Comment by moul** [comment]
LGTM, ping @nledez

File: Dockerfile [hunk 1]
```diff
@@ -47,26 +47,21 @@ RUN apt-get update \
 	python-dateutil python-dev python-distlib python-html5lib python-meld3 \
 	python-mysqldb python-pip python-pkg-resources python-pycurl python-setuptools \
 	python-software-properties python-tornado python-virtualenv python-whoosh \
-	python2.7-dev sqlite3 ssl-cert supervisor zlib1g-dev && \
-	apt-get clean
```
Comment (nledez) [inline Dockerfile]: Why remove this "apt-get clean"?

File: Dockerfile [hunk 2]
```diff
@@ -47,26 +47,21 @@ RUN apt-get update \
...
+RUN wget -O - http://ubuntu.cozycloud.cc/cozy.gpg.key 2>/dev/null | apt-key add - \
+ && echo 'deb http://ubuntu.cozycloud.cc/debian trusty main' > /etc/apt/sources.list.d/cozy.list \
+ && apt-get update \
+ && apt-get install -y --force-yes cozy-indexer \
+ && apt-get install --download-only \
+   couchdb         \
+   couchdb-bin     \
+   couchdb-common  \
+   cozy            \
+ && apt-get clean
```
Comment (nledez) [inline Dockerfile]: And add "apt-get clean" here? The "apt-get install --download-only" is here to have some packages allready on vm. And apt-get clean remove them.

File: Dockerfile [hunk 3]
```diff
@@ -47,26 +47,21 @@ RUN apt-get update \
...
 
-RUN wget -O - http://ubuntu.cozycloud.cc/cozy.gpg.key 2>/dev/null | apt-key add - && \
-	echo 'deb http://ubuntu.cozycloud.cc/debian trusty main' > /etc/apt/sources.list.d/cozy.list && \
-	apt-get update ; \
-	apt-get update && \
-	apt-get install -y --force-yes cozy-indexer && \
-	apt-get install --download-only cozy couchdb couchdb-bin couchdb-common
+RUN wget -O - http://ubuntu.cozycloud.cc/cozy.gpg.key 2>/dev/null | apt-key add - \
+ && echo 'deb http://ubuntu.cozycloud.cc/debian trusty main' > /etc/apt/sources.list.d/cozy.list \
+ && apt-get update \
```
Comment (nledez) [inline Dockerfile]: The "apt-get update ; apt-get update &&" is here because the first time the update fail.

**Comment by QuentinPerez** [comment]
Done

**Comment by QuentinPerez** [comment]
Moved

**Comment by moul** [comment]
@nledez is it better ?

**Comment by nledez** [comment]
@moul yep! Who can merge it?

**Comment by moul** [comment]
@nledez: admins of this team are you (@nledez), @QuentinPerez and me (@moul)

Do not hesitate to merge something as soon as you consider it good enough, then ping me here (mention @moul), so I can start a new build if this is what you want

**Comment by nledez** [comment]
Many thanks to @QuentinPerez.

@moul you can launch built to update public image :)

**Comment by moul** [comment]
@nledez: I fixed the permission issue, can you try to merge this PR ?

Then, I will launch a new build

**Pull request merged**